microsoft/PowerToys
# [FancyZones] Modern apps snapping fix

## Summary of the Pull Request

* Remove the `Allow popup windows snapping` option
* Added unit tests 


## PR Checklist

- [x] **Closes:** #29491, https://github.com/microsoft/PowerToys/issues/16956, https://github.com/microsoft/PowerToys/issues/29240


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed

* Try to snap Calculator with disabled `Allow popup windows snapping`.
* Verify other popup windows behave as expected (Godot menus, Edge menus, start menu are not snapped)
